cms/startup.py

```diff
@@ -9,6 +9,8 @@
 
 from django_startup import autostartup
 from monkey_patch import django_utils_translation
+from util import keyword_substitution
+from lms.startup import get_keyword_function_map
```

Comment:
Yeah, I'm cool with it; more curious about convention than anything, so thanks for the link.